java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -819,6 +819,9 @@ public int hashCode() {
           valueHash = v.hashCode();
         }
         break;
+      case DECIMAL128:
```

Comment:
`hashCode` and `equals` are closely related, and it appears we're now allowing hashing of DECIMAL128 scalars but not allowing them to be tested as equal?

Reply:
Yikes. Good point.